### ISSUE TITLE
Text is incompatible with almond even when inlining text during build

### ISSUE THREAD

**Issue by zero298** [requirejs/text#101]
I set up my require.js project with the text plugin and built with r.js which inlined the text requirement.  However, the `load` method of text depends on `toUrl` which require.js has, but almond does not since it does not support url loading.  However, this resource was inlined so it really is available in the singular compressed file.

```javascript
var parsed = text.parseName(name),
   nonStripName = parsed.moduleName +
   (parsed.ext ? '.' + parsed.ext : ''),
   url = req.toUrl(nonStripName),  // Almond has no support for toUrl
   useXhr = (masterConfig.useXhr) ||
   text.useXhr;
```

Is there a way to utilize text with almond after an r.js build?

**Comment by zero298** [comment]
Actually, I think I may be doing something that the optimizer may not account for.  Either that, or I'm doing something that I may not supposed to be doing.  In my config, I set a path to my resource collection: images, json files, and such.

```javascript
paths: {
    res: "../../res"
}
```

Then in a module, I require that in with text!

```javascript
define([
    "text!res/json/quotes.json"
], function (
        quoteFile
        ) {
    // Do stuff with quoteFile
    return quoteFile;
});
```

When I build, I get an inlined version:

```javascript
define('text/text!res/json/quotes.json',[],function () { return "...";})
````

However, if I don't use the path, everything seems to work correctly.

```javascript
define([
    "text!../../../res/json/quotes.json"
], function (
        quoteFile
        ) {
    // Do stuff with quoteFile
    return quoteFile;
});
```
Produces:
```javascript
define('text/text!namespace/../../../res/json/quotes.json',[],function () { return "...";})
````

Am I not supposed to do this with paths?  Using the path and running the unoptimized version works fine.  But the optimized will resolve to the incorrect path.

**Comment by alberto-acevedo** [comment]
zero298,

Did you find a solution or a workaround  to your issue?. I'm having the same issue.

Thanks,

Alberto

**Comment by zero298** [comment]
@alberto-acevedo No, I don't believe that I did.  I ended up just using the full require.js loader rather than almond.  I was planning on comparing how dojo deals with the text plugin and comparing that to almond to see what the issue was.
